website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -11,6 +11,10 @@ Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
 ## 2023-06-09
 
+* Introduce support for [`import` blocks](/terraform/language/import/generating-configuration).
```

Comment:
I'm a little late getting these docs in, but June 9 was the first day these new API attributes were available in Terraform Cloud. Is that the correct date to put on the changelog?